src/disco/constants.js

```diff
@@ -0,0 +1,6 @@
+export const DOWNLOADING = 'downloading';
```

Comment:
I think these should be namespaced somehow, either just putting them on an `installStates` object or something nice with modules. Once we start adding more constants it's probably going to be convenient to grab all constants of a certain type.